.github/workflows/checkbox-core-snap-beta-release.yml

```
@@ -50,6 +51,25 @@ jobs:
         with:
           name: series${{ matrix.releases }}
           path: checkbox-core-snap/series${{ matrix.releases }}/*.snap
+      - name: Check that snapcraft built the expected amount of snaps
+        run: |
+          NB_SNAPS=$(ls -l checkbox-core-snap/series${{ matrix.releases }}/*.snap | wc -l)
```

You shouldn't parse the output of ls(1) (See http://mywiki.wooledge.org/ParsingLs)

```suggestion
          NB_SNAPS=$(find checkbox-core-snap/series${{ matrix.releases }} -maxdepth 1 -type f -name *.snap | wc -l)
```

---

Thanks! I made the modification both for the checkbox-core and the checkbox snap workflows.